source/give.html.erb

```diff
@@ -0,0 +1,75 @@
+---
+title: Give
+---
+
+<h1>Giving to Grafeas</h1>
+
+<p>We at Grafeas are passionate about making the world a better place, but we can't do it without your help. Our volunteers work diligently around the clock to help us get closer to this goal, but we still have operating costs that hard work cannot cover.</p>
+
+<p>Your donation to the Grafeas Group will help us further our goal of worldwide accessibility on the internet. All funding goes towards:</p>
+
+<p>
+<li>Upgrading servers</li>
+<li>Community management</li>
+<li>Outreach</li>
+<li>Forging new partnerships</li>
+</p>
+
+<p>Every dollar counts; we recommend picking the option that's best for you &mdash; if the pre-filled options don't suit your needs, please use the "Other" button. NOTE: IMPLEMENTING THE OTHER BUTTON IS A PAIN IN THE BUTT</p>
```

Comment:
I'd recommend changing the wording here to something like "reach out to us if one of the pre-filled options doesn't suit your needs", then we can add an "Other" button later. By that point, "Other" will be pretty self-explanatory and we won't even need this paragraph.